widgets/nuxeo-file.html

```diff
@@ -67,19 +91,37 @@
       </dom-if>
     </div>
 
-    <dom-repeat items="[[files]]" as="file">
+    <label class="error" hidden$="[[!invalid]]">[[errorMessage]]</label>
+
+    <dom-if if="[[!multiple]]">
+      <template>
+        <dom-if if="[[value]]">
+          <template>
+            <div class="file">
+              <div class="layout horizontal">
+                <a href$="[[value.data]]">[[_fileName(value)]]</a>
```

Comment:
This `href` will not work for uploaded blobs. In these cases, we could either:
1) just use a label instead, or
2) fill the data property with a computed url for the file we just uploaded, as reference by the browser

I think the second approach is preferable. See implementation suggestion in the comments bellow. ⬇️